relayer/crates/starknet-integration-tests/src/tests/erc20.rs

```diff
@@ -141,7 +186,6 @@ fn test_erc20_transfer() -> Result<(), Error> {
 
             match &erc20_events[0] {
                 Erc20Event::Transfer(transfer) => {
-                    assert_eq!(transfer.from, account_address);
```

Comment:
why this line is deleted ?

Reply:
We were not sending tokens from user 1 to user 2, but from relayer wallet to another address.
Since we now use multiple relayer wallets this assertion isn't correct anymore

Reply:
hmm. can we use `send_messaged_with_signer` to keep these asserts ?